### ISSUE TITLE
Navigation/A11y: Can still tab through navigation items when SectionNav collapsed

### ISSUE THREAD

**Issue by JoaoSilvaGrafana** [grafana/grafana#58230]
After collapsing the section navigation, you can still tab through the items that are inside that navigation even though they are not visible:

(in the screen grab I am pressing Shift+Tab and even though you can't see it move you can see the links change at the bottom)

https://user-images.githubusercontent.com/100691367/199972606-3ff12c06-f22c-4a95-ae84-a58c3d4e038a.mov
